### PR TITLE
Add patch cherrypick process

### PR DESCRIPTION
**Pull request by cfkoehler**

### CODE REVIEW

File: RELEASING.md [hunk 1]
```diff
@@ -73,6 +73,8 @@ Options:
 Releases patch fixes from a branch called `patch/<version>`, i.e. `patch/8.0.x`. A release branch called `release/<version>` is created
 and the release is performed. When finished, the patch branch is deleted. No commits are made to `master`.
 
+Changes that are in a patch branch should first have a pull request that targets the Master/Main branch and once approved cherrypicked to the patch branch for release.
```
[the remark this answers is not in the document]
Comment (cfkoehler) [reply RELEASING.md:76]: Ah. good point. I think that is referring to no maven release pom changes are added to master. Maybe?

I will try to reword this section. But I would expect that a patch would be needed to fix a bug and that we would want that bug fix to also be part of the baseline. That is what I was trying to say

**Review by ldhardy — APPROVED** [review]
I know there are multiple schools of thought on how to handle patching previous releases, so being clear about how we'd like to do it is important.  Thanks for adding this.  

Do you think that there should be two PRs? One for main branch for the actual substance of the patch and one for the cherrypicks for the patch branch?   Because cherrypicking is done relatively infrequently, people (I am people) sometimes make mistakes with the git commands and verification so it's nice to have another set of eyes on the cherry pick commit.

**Comment by cfkoehler** [comment]
I am not sure on the standard practice. But what if a github action did the cherry pick to a patch branch. Given x commit hash put to y path version

**Comment by ldhardy** [comment]
> I am not sure on the standard practice. But what if a github action did the cherry pick to a patch branch. Given x commit hash put to y path version

That would be cool.  Anything that will cut down on human errors.  I'm not saying the Github actions would always be able to do such a merge without supervision.  But probably in most cases this would be the most consistent way to handle a patch.

**Comment by dev-mlb** [comment]
> I am not sure on the standard practice. But what if a github action did the cherry pick to a patch branch. Given x commit hash put to y path version

How about this: https://github.com/NationalSecurityAgency/emissary/pull/537?

**Comment by cfkoehler** [comment]
Closing in favor of doc updates in https://github.com/NationalSecurityAgency/emissary/pull/537

**Pull request closed without merging**